proxy/http2/HPACK.cc

```diff
@@ -228,12 +266,12 @@ namespace HpackStaticTable
     HpackLookupResult result;
 
     for (unsigned int index = 1; index < TS_HPACK_STATIC_TABLE_ENTRY_NUM; ++index) {
-      std::string_view name  = STATIC_TABLE[index].name;
-      std::string_view value = STATIC_TABLE[index].value;
+      const std::string_view &name  = STATIC_TABLE[index].name;
+      const std::string_view &value = STATIC_TABLE[index].value;
```

Comment:
Is the const reference cheaper than copying the pointer and length values of original `name` and `value` types, as the original did? Each time these are accessed, it will now require a pointer dereference.

Reply:
This also depends on how it'll be optimized probably. I don't know why but I saw memcpy without this change. I presume, by having these as const references, they are handled as aliases (no copies but direct access to the name, and the value).

Reply:
That is really strange. I wouldn't expect the creation of a string view to have a memcpy...a string_view is simply nothing more than a pointer to a char* and a length (int type) value.

Reply:
In any case, if your profiling found that const references are more performant, let's run with that. Can you also add a comment here explaining that the const references were more performant than std::string_view copies?